src/components/projects/CreateProjectForm.vue

```diff
@@ -322,12 +322,10 @@
                 size="is-small"
                 :disabled="!exposures[n-1].active || read_only"
                 type="number"
-                min="0"
+                min="0.000001"
```

Comment:
This is fine. Honestly in practice a min of 0.1 with step of 0.1 would probably suffice. I don't know that any of these cameras can control the exposure time in microseconds lol

Reply:
But doing what Michael said is totally fine, just go with that.